src/index.js

```diff
@@ -28,26 +29,24 @@ export const actions = withHandlers({
   decrement: ({ setCounter }) => () => setCounter(n => n - 1),
 });
 
+export const counterStyle = Style.of({
+  flex: 1,
+  justifyContent: "center",
+  alignItems: "center",
+});
+
 export const Counter = (
   {
     increment,
     counter,
     decrement,
   }: IWithHandlers,
 ) => (
-  <View style={style.counter}>
+  <View style={counterStyle.resolve()}>
```

Comment:
@JakeDawkins I think we should PR adding memoization into further for the resolve function. Same props shouldn't need to recalc the return object each time.

Not for this PR, but just wanted to mention it